clash-lib/src/Clash/Backend/VHDL.hs

```diff
@@ -1133,6 +1179,11 @@ tyName' rec0 (filterTransparent -> t) = do
                             , if rec0 then showt n `TextS.append` "_" else ""
                             , elTy']
       Ap $ makeCached (t, rec0) nameCache (return nm)
+    MemBlob n m -> do
+      return $ TextS.concat [ "memblob_of_"
+                            , if rec0 then showt n `TextS.append` "_" else ""
+                            , "std_logic_vector_"
+                            , showt m]
```

Comment:
This can be removed. And I would add the following clause to `filterTransparent` https://github.com/clash-lang/clash-compiler/blob/d087788ded8721c60a912b05015bdb28e5d36aaa/clash-lib/src/Clash/Backend/VHDL.hs#L1210
```
  MemBlob n m -> Vector n (BitVector m)
```